### PR TITLE
redirect /_plugins/crate-admin to root

### PR DESCRIPTION
**Pull request by m-kharbat**

### CODE REVIEW

File: admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java [hunk 1]
```diff
@@ -56,6 +56,27 @@ public void testLegacyRedirect() throws IOException, URISyntaxException {
...
+    public void testPluginURLRedirect() throws IOException, URISyntaxException {
+        //request to '/_plugin/crate-admin' is redirected to '/index.html'
+        Header[] headers = {
+            browserHeader()
+        };
+
+        List<URI> allRedirectLocations = getAllRedirectLocations("/_plugin/crate-admin", headers);
+
+        // all redirect locations should not be null
+        assertThat(allRedirectLocations, notNullValue());
```
Comment (seut) [inline admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java]: remove, unnecessary, contains assertion will fail if its null, it's just confusing ;)

File: admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java [hunk 2]
```diff
@@ -56,6 +56,27 @@ public void testLegacyRedirect() throws IOException, URISyntaxException {
...
+        // all redirect locations should contain the crateAdminUI URI
+        assertThat(allRedirectLocations.contains(adminURI()), is(true));
+    }
+
+    @Test
+    public void testPluginURLRedirectReturnsIndex() throws IOException, URISyntaxException {
+        //request to '/_plugins/crate-admin' is redirected to '/index.html'
+        assertIsIndexResponse(browserGet("/_plugin/crate-admin"));
```
Comment (seut) [inline admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java]: is this test really different than the above? if so pls explain, to me it looks like testing the same using the same internal calls...

Comment (m-kharbat) [inline admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java]: /_plugins/crate-admin => redirects to / => responds with index.html

@seut the first test checks if /_plugins/crate-admin redirects to /. 
The second tests if requesting /_plugins/crate-admin responds with  index.html

Comment (seut) [inline admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java]: @m-kharbat so the comment at the first test is wrong?

Comment (m-kharbat) [inline admin-ui/src/test/java/io.crate/rest.action.admin/AdminUIIntegrationTest.java]: @seut fixed the comment and squashed all commits

File: admin-ui/src/main/java/io/crate/rest/action/admin/AdminUIStaticFileRequestFilter.java [hunk 3]
```diff
@@ -62,6 +62,12 @@ public AdminUIStaticFileRequestFilter(Environment environment) {
 
     @Override
     public void process(RestRequest request, RestChannel channel, RestFilterChain filterChain) throws IOException {
+        if (request.rawPath().equals("/_plugin/crate-admin/")){
```
Comment (seut) [inline admin-ui/src/main/java/io/crate/rest/action/admin/AdminUIStaticFileRequestFilter.java]: this won't catch `/_plugin/crate-admin/index.html`

**Pull request merged**